coding-concepts.js

```diff
@@ -51,9 +70,19 @@ class Learn {
     this.cohort = "Charlie",
     this.year = 2021
   }
+
+  hello(){
+    return `Hello`
+  }
 }
 var learnStudent = new Learn
-// console.log(learnStudent)
-
-// a) Your answer:
-// b) Verify and explain:
+console.log(learnStudent)
+
+// a) Your answer:Learn { student: 'George', cohort: 'Charlie', year: 2021 }
+// b) Verify and explain: output as expected: Learn { student: 'George', cohort: 'Charlie', year: 2021 }
+// The output will be all attributes(data) because of console.log(learnStudent). Here use "new" to create an instance of "Learn".  
+// Try different output to see how it works.
```

Comment:
Amazing!

Reply:
Thank you for your all comments and hard work.